aloha/real_env.py

```diff
@@ -85,33 +83,48 @@ def __init__(
             True. Only applies when IS_MOBILE is True
         :raises ValueError: On providing False for setup_base but the robot is not mobile
         """
-        self.follower_bot_left = InterbotixManipulatorXS(
-            robot_model='vx300s',
-            group_name='arm',
-            gripper_name='gripper',
-            robot_name='follower_left',
-            node=node,
-            iterative_update_fk=False,
-        )
-        self.follower_bot_right = InterbotixManipulatorXS(
-            robot_model='vx300s',
-            group_name='arm',
-            gripper_name='gripper',
-            robot_name='follower_right',
-            node=node,
-            iterative_update_fk=False,
-        )
-
-        self.recorder_left = Recorder('left', node=node)
-        self.recorder_right = Recorder('right', node=node)
-        self.image_recorder = ImageRecorder(node=node, is_mobile=IS_MOBILE)
+        self.is_mobile = config.get('base',False)
```

Comment:
Make sure to lint these files

```suggestion
        self.is_mobile = config.get('base', False)
```